pages/common/gradle.md

```diff
@@ -1,15 +1,27 @@
 # gradle
 
-> Gradle is the official build system for Android Studio.
+> Gradle is an advanced open source build automation system.
 
 - Compile a package:
 
 `gradle build`
 
+- Exclude test task:
+
+`gradle build -x test`
+
+- Run in offline mode:
+
+`gradle build --offline`
+
 - Clear the build folder:
 
 `gradle clean`
 
 - Compile and Release package:
 
 `gradle assembleRelease`
+
+- Create Gradle wrapper:
```

Comment:
We have to also explain the options used in the command. Either change the description to `Create gradle wrapper with a given gradle version`, or change the example to exclude specifying the gradle-version. 

Remember to always focus on the most common and practical examples. If specifying the gradle-version is not a common thing, then we can drop it.

Reply:
I've removed that example. It seems not to be a very common use case.